frontend/src/components/assessments/assessment-creation/StudentDashboardHeader.tsx

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import { Box, HStack, Text, VStack } from "@chakra-ui/react";
+
+import HeaderWrapper from "../../common/HeaderWrapper";
+
+interface StudentDashboardHeaderProps {
+  assessmentName: string;
+  classroomName: string;
+  estimatedTime: string;
```

Comment:
can you check w/ design if `estimatedTime` is still needed in the header? i think they might have made some changes since the ticket was created.

riishi is the designer to reach out to!

Reply:
yep they are removing it